language/de/captcha_recaptcha.php

```diff
@@ -46,10 +46,10 @@
 	'RECAPTCHA_INCORRECT'			=> 'Die von dir eingegebene Antwort ist falsch',
 	'RECAPTCHA_NOSCRIPT'			=> 'Bitte aktiviere JavaScript in deinem Browser, um die Aufgabe zu laden.',
 
-	'RECAPTCHA_PUBLIC'				=> 'Öffentlicher reCaptcha-Schlüssel',
-	'RECAPTCHA_PUBLIC_EXPLAIN'		=> 'Dein öffentlicher reCaptcha-Schlüssel. Schlüssel können über <a href="http://www.google.com/recaptcha">www.google.com/recaptcha</a> bezogen werden.',
-	'RECAPTCHA_PRIVATE'				=> 'Privater reCaptcha-Schlüssel',
-	'RECAPTCHA_PRIVATE_EXPLAIN'		=> 'Dein privater reCaptcha-Schlüssel. Schlüssel können über <a href="http://www.google.com/recaptcha">www.google.com/recaptcha</a> bezogen werden.',
+	'RECAPTCHA_PUBLIC'				=> 'Seiten-Schlüssel',
+	'RECAPTCHA_PUBLIC_EXPLAIN'		=> 'Der öffentliche reCaptcha-Schlüssel für deine Seite. Schlüssel können über <a href="http://www.google.com/recaptcha">www.google.com/recaptcha</a> bezogen werden. Bitte verwende das unsichtbare reCAPTCHA in der Version 2 (Typ: reCAPTCHA v2 &gt; Invisible reCAPTCHA badge).',
```

Comment:
Im Original steht da nicht von "öffentlich", sondern ist der neuen Version nur noch "Your site reCAPTCHA key."

Reply:
Google verwendet "Websiteschlüssel" und "Geheimer Schlüssel" - so würde ich das hier auch machen

Reply:
Ferner: Bitte verwende das unsichtbare reCAPTCHA-Logo in der Version 2

Reply:
Wo hast du das von Google denn eigentlich her? Ich habe ehrlich gesagt immer nur was Englisches angezeigt bekommen. Ich finde Öffentlich und Privat viel eingänglicher als Seitenschlüssel und Privat. Deshalb habe ich die Änderung aus dem Englischen auch nicht übertragen. Aber meinetwegen nehmen wir das gleiche wie Google, dann findet man es wenigstens.

Reply:
Hab's auch nicht direkt bei Google gefunden, da ich nicht in den Registrierungsprozess einsteigen wollte. Aber alle Anleitungen beziehen sich immer auf "Websiteschlüssel" - z. B. https://www.webdesign-podcast.de/2016/11/23/google-recaptcha-einbinden-und-spam-abwehren/